l2geth/core/types/transaction_meta.go

```diff
@@ -32,6 +32,17 @@ func (q QueueOrigin) String() string {
 	}
 }
 
+func (q QueueOrigin) MarshalJSON() ([]byte, error) {
+	switch q {
+	case QueueOriginSequencer:
+		return []byte("\"sequencer\""), nil
```

Comment:
Could you get rid of the need to the escapes by using a backtick string?

Reply:
certainly. thanks for pointing that out.